errolyn/a10.py

```diff
@@ -0,0 +1,35 @@
+
+bit_string = input("Please enter a 32-bit integer: ") # example: 10000000111111110000001100001111
```

Comment:
It's common to put function definitions before any logic in the main flow of the program; I consider taking user input to be part of that main flow and would expect it with the code you put in `handler` instead of here.